packages/@o3r/workspace/schematics/ng-add/helpers/npm-workspace.ts

```diff
@@ -75,3 +75,33 @@ export function filterPackageJsonScripts(tree: Tree, _context: SchematicContext)
   tree.overwrite(rootPackageJsonPath, JSON.stringify(rootPackageJsonObject, null, 2));
   return tree;
 }
+
+/**
+ * Add lerna at the root of the project
+ * @param o3rWorkspacePackageJson the @o3r/workspace package.json
+ */
+export function addLerna(o3rWorkspacePackageJson: PackageJson & { generatorDependencies: Record<string, string> }): Rule {
+  return (tree: Tree, _context: SchematicContext) => {
+    const rootPackageJsonPath = '/package.json';
+    if (!tree.exists(rootPackageJsonPath)) {
+      throw new SchematicsException('Root package.json does not exist');
+    }
+    const rootPackageJsonObject = tree.readJson(rootPackageJsonPath) as PackageJson;
+    rootPackageJsonObject.devDependencies = {
+      ...rootPackageJsonObject.devDependencies,
+      'lerna': o3rWorkspacePackageJson.generatorDependencies.lerna
+    };
+    rootPackageJsonObject.scripts = {
+      ...rootPackageJsonObject.scripts,
+      'build': 'lerna run build',
+      'test': 'lerna run test',
+      'lint': 'lerna run lint'
+    };
+    tree.create('/lerna.json', JSON.stringify({
```

Comment:
would not make sense to run `npx lerna init` instead?

Reply:
I tried doing a lerna init but since the tree is in memory, and the `addWorkspacesToProject` task is not executed on the drive, `lerna init` throws an error:
`lerna ERR! Cannot initialize lerna because your package manager has not been configured to use `workspaces`, and you have not explicitly specified any packages to operate on`

Let's stick with adding the dev dep and creating the `lerna.json` by hand since we modify both properties inside ($schema and version).